### PR TITLE
Update to honeybadger 2.0

### PR DESCRIPTION
**Pull request by philbogle**

### CODE REVIEW

**Comment by philbogle** [comment]
These changes are currently being canaried on "frontend--d4".

File: dashboard/config/honeybadger.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+api_key: "#{CDO.dashboard_honeybadger_api_key}"
+
+exceptions.ignore:
+  <%  %w(Sinatra StorageApps DynamoTable PropertyBag).each do |class_name| %>
+  - <%= class_name + "::NotFound" %>
+  <% end %>
```
Comment (davidsbailey) [inline dashboard/config/honeybadger.yml:7]: Why are we ignoring these exceptions? Do we want to stop ignoring them at some point?

**Comment by davidsbailey** [comment]
LGTM

File: dashboard/config/honeybadger.yml (hunk 1 above)
Comment (philbogle) [inline dashboard/config/honeybadger.yml:7]: I don't know.  This is the existing behavior migrated from honeybadger.rb by the config migration script.

**Pull request closed without merging**